src/Core/Domain/AttributeGroup/Attribute/Command/AddAttributeCommand.php

```diff
@@ -62,25 +62,20 @@ class AddAttributeCommand
     private $pathName;
 
     /**
-     * @param int $attributeGroupId
-     * @param array $localizedValue
-     * @param string $color
+     * @param array $localizedNames
      * @param int[] $associatedShopIds
      *
      * @throws AttributeConstraintException
      */
-    public function __construct(int $attributeGroupId, array $localizedValue, string $color, array $associatedShopIds = [])
+    public function __construct(int $attributeGroupId, array $localizedNames, string $color, array $associatedShopIds = [])
```

Comment:
I don't think you need to change this, you can configure the mapping on the module side via `CQRSCommandMapping`